easybuild/easyconfigs/d/DeepLabCut/DeepLabCut-2.3.1-foss-2021b-CUDA-11.4.1.eb

```diff
@@ -0,0 +1,82 @@
+# Loosely based on PR #7680  
+# J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'PythonBundle'
+
+name = 'DeepLabCut'
+version = '2.3.1'
+versionsuffix = '-CUDA-%(cudaver)s'
+local_version = '%(versionsuffix)s-contrib'
+
+homepage = 'http://www.mousemotorlab.org/deeplabcut'
+description = "Markerless tracking of user-defined features with deep learning"
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+toolchainopts = {'pic': True}
+
+builddependencies = [
+    ('pkgconf', '1.8.0'),
+]
+
+dependencies = [
+    ('CUDA', '11.4.1', '', SYSTEM),
+    ('cuDNN', '8.2.2.26', '-CUDA-%(cudaver)s', SYSTEM),
+    ('Python', '3.9.6'),
+    ('h5py', '3.6.0'),
+    ('IPython', '7.26.0'),
+    ('scikit-learn', '1.0.2'),
+    ('scikit-image', '0.19.1'),
+    ('TensorFlow', '2.7.1', versionsuffix),
+    ('PyYAML', '5.4.1'),
+    ('wxPython', '4.2.0'),
+    ('OpenCV', '4.5.5', local_version),
+    ('numba', '0.54.1'),
+    ('libyaml', '0.2.5'),
+    ('imgaug', '0.4.0', versionsuffix),
+    ('tqdm', '4.62.3'),
+    ('ruamel.yaml', '0.17.21'),
+    ('statsmodels', '0.13.1'),
+    ('PyTorch', '1.11.0', versionsuffix),
+    ('MoviePy', '1.0.3'),
+    ('SciPy-bundle', '2021.10'),
+    ('HDF5', '1.12.1'),
+    ('LZO', '2.10'),
+    ('Blosc', '1.21.1'),
+    ('py-cpuinfo', '9.0.0'),
```

Comment:
Some of these are dependencies for `PyTables`, are they also direct dependencies for `DeepLabCut` itself?